test/simplify_reshapes_test.cpp

```diff
@@ -707,20 +707,21 @@ TEST_CASE(optimize_where_true)
         return m;
     };
 
-    auto create_opt_module = [&](std::string name) {
+    auto create_opt_module = [&](bool cond) {
```

Comment:
Maybe name this `return_x` or something?

Reply:
renamed